fsgrid.hpp

```diff
@@ -348,7 +448,7 @@ template <typename T, int stencil> class FsGrid : public FsGridTools{
                      if(storageSize[i] == 1) 
                         subarrayStart[i] = 0;
 
-                  std::array<int,3> swappedStorageSize = storageSize;
+                  std::array<int,3> swappedStorageSize = {(int)storageSize[0],(int)storageSize[1],(int)storageSize[2]};
```

Comment:
So this makes an actual difference beyond making the type conversion extra clear??

Reply:
Gets rid of compile warnings (storageSize is of type FsSize_t)